images/base/persistent_volume_scrubber.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#
+# scrub_directory performs an 'rm -rf' on a directory to scrub it clean
+# before re-use by another cluster resource.
+# this script is intended to be used in a pod that performs the scrub.
+# the container in the pod should succeed or fail based on the exit status of this script.
+#
+function scrub_directory(){
+
+    # first and only arg is the directory to scrub
+    dir=$1
+
+    if [ -z "$dir" ]; then
+        echo "Usage:  scrub_directory some/path/to/scrub"
+        return 1
+    fi
+
+    if test ! -e $dir; then
```

Comment:
Style looks odd to me (but I'm no bash guru)... any advantage over the more common `if [ ! -d "${dir}" ]`?